src/main/java/net/clgd/ccemux/plugins/builtin/JFXPlugin.java

```diff
@@ -0,0 +1,43 @@
+package net.clgd.ccemux.plugins.builtin;
+
+import java.util.*;
+
+import com.google.auto.service.AutoService;
+
+import net.clgd.ccemux.emulation.EmuConfig;
+import net.clgd.ccemux.plugins.Plugin;
+import net.clgd.ccemux.rendering.RendererFactory;
+import net.clgd.ccemux.rendering.javafx.JFXRendererFactory;
+
+@AutoService(Plugin.class)
+public class JFXPlugin extends Plugin {
+	@Override
+	public String getName() {
+		return "JavaFX Renderer";
+	}
+
+	@Override
+	public String getDescription() {
+		return "A renderer using JavaFX";
+	}
+
+	@Override
+	public Optional<String> getVersion() {
+		return Optional.empty();
+	}
+
+	@Override
+	public Collection<String> getAuthors() {
+		return Collections.singleton("CLGD");
```

Comment:
This probably belongs in its own issue, but since we're in the CCEmuX organisation now, perhaps "CCEmuX Team" or similar should be the author instead?

Reply:
Yeah, that would probably be a good idea